app/full_text_search.py

```diff
@@ -42,25 +73,40 @@ def search(repo, query_txt, max_results):
     results = []
     if not query_txt:
         return results
```

Comment:
Maybe this is the only place where "results" is used? If so, just return [] instead.